website/docs/en/guide/start/introduction.mdx

```diff
@@ -57,11 +57,11 @@ Rspack was initially created to solve performance problems encountered at ByteDa
 
 ## Current status of Rspack
 
-Rspack has been under continuous development since April 2022, and was launched as open source in March 2023. Rspack has already implemented most important features of webpack and these features can meet the needs of most projects. Some ByteDance internal projects have already achieved a 5-10 times improvement moving from webpack to Rspack, and there's still room for future optimizations.
+In August 2024, we released [Rspack 1.0](/blog/announcing-1-0), which is is production-ready and covers most of webpack's APIs and features.
 
-Rspack is compatible with webpack's configuration schema and loader architecture. You can seamlessly use familiar loaders such as `babel-loader`, `less-loader`, `sass-loader`, `vue-loader`, etc.
+Rspack is currently compatible with almost all loaders in the community. For the 50 most downloaded webpack plugins, more than 80% can be used in Rspack or have an alternative.
```

Comment:
`Rspack is currently compatible with almost all loaders in the community.` (maybe include a list of whats not compatible?)

`For the 50 most downloaded webpack plugins, more than 80% can be used in Rspack or have an alternative.` (also include the list of supported and which have alternatives)

Reply:
> maybe include a list of whats not compatible

We have not found any incompatible loaders yet. If a loader accesses the compilation object internally, it may be incompatible.

> include the list of supported and which have alternatives

Link added